examples/with-fastify/index.js

```diff
@@ -64,15 +81,14 @@ const run = async () => {
     });
   });
 
-  await app.listen({ port: 3000 });
+  await app.listen({ host: serverOptions.host, port: serverOptions.port });
   // eslint-disable-next-line no-console
-  console.log('Running on 3000...');
-  console.log('For the UI, open http://localhost:3000/ui');
-  console.log('Make sure Redis is running on port 6379 by default');
+  console.log(`For the UI, open http://localhost:3333/ui`);
+  console.log('Make sure Redis is configured in env variables. See .env.example');
   console.log('To populate the queue, run:');
-  console.log('  curl http://localhost:3000/add?title=Example');
+  console.log('  curl http://localhost:3333/add?title=Example');
```

Comment:
it should use the serverOptiosn.host & port, no?

Reply:
Good challenge!

Sorry - an oversight, the exposed port is configured in docker-compose.yml, which is pinned to 3333 in config.

I should remove serverOptions and hardcode to 0.0.0.0 and 3000 as this is implicit in compose config

Reply:
Latest commit addresses the issue

Reply:
I still see usage of `3333` port, why?

Reply:
Port `3000` is an arbitrary albeit conventional port for nodejs applications. From my perspective moving the exposed port to anything is a reasonable thing to do, especially where the primary application (the one posting to or subscribing to the MQ is likely going to reside on that port.  With bull-board, as pretty as it is, being an ancillary support app, it seems to be a logical move.

If you would like me to revert to `3000` I can, but obvs I'll be running on `3333` locally.